src/FormField/ScopeBuilder.php

```diff
@@ -524,16 +543,16 @@ public static function conditionToQuery(Condition $condition): array
 
             $map = [
                 'LIKE' => [
-                    'equals',
-                    'begins with',
-                    'ends with',
-                    'contains'
+                    self::OPERATOR_EQUALS,
+                    self::OPERATOR_BEGINS,
+                    self::OPERATOR_ENDS,
```

Comment:
`ENDS_WITH` sound much more natural

Reply:
Done now, I am a proponent of good semantics in code

Reply:
Great!